### ISSUE TITLE
Do three-globe support tree-shaking for webpack?

### ISSUE THREAD

**Issue by KevinZhang19870314** [vasturiano/three-globe#61]
I am using your awesome library in my project, but then I found that the bunder size is a little bit large, I am not sure thee-globe can support tree-shaking or not?

`import ThreeGlobe from "three-globe";`

![image](https://user-images.githubusercontent.com/13794286/185370987-5d01cba1-adb1-440b-8cda-9388b757b268.png)

**Comment by vasturiano** [comment]
@KevinZhang19870314 yes the module should be tree-shakeable. More info about it here:
https://bundlephobia.com/package/three-globe

**Comment by KevinZhang19870314** [comment]
> @KevinZhang19870314 yes the module should be tree-shakeable. More info about it here: https://bundlephobia.com/package/three-globe

Okay, so it means three-globe bundler size is normal and also has been tree shaked? If is it, then thank you, I'll take my time to look into other packages size in my bundler.
